sema/subyt/sources.py

```
@@ -20,8 +20,10 @@
 
 def assert_readable(path_name: str | Path):
     in_path = Path(path_name)
-    assert in_path.is_file(), f"File to read '{path_name}' does not exist"
-    assert os.access(in_path, os.R_OK), f"Can not read '{path_name}'"
+    if not in_path.is_file():
+        raise ValueError(f"File to read '{path_name}' does not exist")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)